### ISSUE TITLE
Request for verification

### ISSUE THREAD

**Issue by burlexpo** [bethlakshmi/GBE2#315]
I've just scheduled all the Reg desk shifts, including set-up on Thursday evening and Friday morning.

I *think* I've done it right, and, if I have, I *really* like this tool.

Can someone  (@jonkiparsky) take a look at what I did for general tech integrity?

@bethlakshmi can you take a look and see if you agree with how I set up the time slots?

Thanks! I could use this feedback soon, since I'd like to set up more 'generic' volunteer slots.

**Comment by jonkiparsky** [comment]
Looks good so far. I see complete coverage of the registration desk area
for the duration of the event, which seems right. Four or five hours seems
like a long shift, but I assume you find people to work those.
The fact that all instances of an event share an underlying duration is
probably not ideal in this case, but nothing to be done about that now.
Possible improvement for future: store default duration on the conference
model, and allow the scheduler item to override that default.
I'll look a little more, but first pass looks good.

**Comment by burlexpo** [comment]
~4 hours is our standard shift.  Less time isn't really worth it to schedule someone in terms of management complexity.

"The fact that all instances of an event share an underlying duration is probably not ideal in this case, but nothing to be done about that now."

Not a big deal, since I create the first shift as 4.5 hours and then use the duplicate function (which overrides the 76 hour duration from the initial event).

**Comment by bethlakshmi** [comment]
Made 2 minor changes;
- I will pull and all nighter if I have to, to get my info aligned for volunteers on Fri Morning.  I don't want bright eyed people on my doorstep at 9:00 AM.  I made it 10:30.
- I moved Sunday night reg to 7:30 - 8:00 is a bad time for volunteers to wander in, I want the overlap before the Sunday Night Showcase kicks off.

Otherwise - thumbs up.  When brain is more working I will make some notes on visual things I'd like to change next year upon seeing this much data floating in front of my eyes.
